.github/workflows/docker-build-test.yaml

```diff
@@ -302,7 +302,7 @@ jobs:
     with:
       GIT_SHA: ${{ needs.determine-docker-build-metadata.outputs.gitSha }}
       FORGE_TEST_SUITE: compat
-      IMAGE_TAG: testnet # test against a previous testnet release
```

Comment:
let's leave testnet

Reply:
@perryjrandall

Reply:
Actually that will not work when we update testnet with 1.10
 This needs to be 1.9.5...